be/src/common/config.h

```diff
@@ -314,6 +314,7 @@ CONF_Int64(load_data_reserve_hours, "4");
 // log error log will be removed after this time
 CONF_mInt64(load_error_log_reserve_hours, "48");
 CONF_Int32(number_tablet_writer_threads, "16");
+CONF_Int32(number_tablet_sink_chunk_split_threads, "16");
```

Comment:
The default behavior should not be changed.

Reply:
introduced another setting - enable_async_tablet_sink_chunk_split_executor, disable this feature by default